src/routes/package-create.js

```diff
@@ -52,7 +52,7 @@ class PackageCreateRoute extends Component {
 
   render() {
     const { history, location } = this.props;
-    let onCancel = null;
+    let onCancel = () => {};
```

Comment:
If `onCancel` is not required, having this be `undefined` should work. `let onCancel;`

Reply:
hmmmmm still getting the prop type warning. I might just switch `onCancel` to not be required.

Reply:
Oh yea. If it's required it should be a no-op. I think un-requiring is better than a no-op though